test/e2e/pods.go

```diff
@@ -270,6 +270,12 @@ var _ = Describe("Pods", func() {
 		if !deleted {
 			Fail("Failed to observe pod deletion")
 		}
+
+		pods, err = podClient.List(labels.SelectorFromSet(labels.Set(map[string]string{"time": value})), fields.Everything())
+		if err != nil {
+			Fail(fmt.Sprintf("Failed to delete pod: %v", err))
+		}
+		Expect(len(pods.Items)).To(Equal(0))
```

Comment:
When graceful deletion lands this still needs to be empty.  In graceful deletion there are more ways for deletion to not really be deletion.